### PR TITLE
Make left/top auto value consistent across browsers

### PR DESCRIPTION
**Pull request by yiminghe**
http://bugs.jquery.com/ticket/13767

tests:

 http://jsfiddle.net/yiminghe/Yu9sP/5/

### CODE REVIEW

File: test/unit/css.js [hunk 1]
```diff
@@ -874,6 +874,19 @@ test( "css opacity consistency across browsers (#12685)", function() {
...
+			"<div style='position: absolute'></div><span></span><s style='position: fixed'></s></div>").appendTo(fixture);
+
+	equal( el.css("left"), "0px" );
+	equal( el.find("div").css("top"), "20px" );
+	equal( el.find("span").css("top"), "auto" );
+	notEqual( el.find("s").css("top").indexOf("px"), -1 );
+});
+
 test( ":visible/:hidden selectors", function() {
 	expect( 13 );
```
Comment (john-newton) [inline test/unit/css.js]: Expect 13

File: src/css.js [hunk 2]
```diff
@@ -506,9 +506,16 @@ jQuery(function() {
 			jQuery.cssHooks[ prop ] = {
 				get: function( elem, computed ) {
 					if ( computed ) {
-						computed = curCSS( elem, prop );
-						// if curCSS returns percentage, fallback to offset
-						return rnumnonpx.test( computed ) ?
+						var isAutoPosition,
+							elStyles = getStyles( elem ),
```
Comment (a125429812) [inline src/css.js]: 4

Comment (AlexandreDC) [inline src/css.js]: 5

Comment (ketangodhani) [inline src/css.js]: ">

File: src/css.js [hunk 3]
```diff
@@ -506,9 +506,16 @@ jQuery(function() {
...
+							position = curCSS( elem, "position", elStyles );
+						computed = curCSS( elem, prop, elStyles );
+						isAutoPosition = computed === "auto";
```
Comment (fugui) [inline src/css.js]: It is too difficult to read, so you need break to two lines .

Comment (fugui) [inline src/css.js]: and add another line to comment it.

File: src/support.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ jQuery.support = (function( support ) {
 
 	// Run tests that need a body at doc ready
 	jQuery(function() {
-		var container, marginDiv,
```
Comment (305029312) [inline src/support.js]: sdffsdf

File: src/css.js [hunk 5]
```diff
@@ -506,9 +506,16 @@ jQuery(function() {
 			jQuery.cssHooks[ prop ] = {
 				get: function( elem, computed ) {
 					if ( computed ) {
-						computed = curCSS( elem, prop );
-						// if curCSS returns percentage, fallback to offset
-						return rnumnonpx.test( computed ) ?
+						var isAutoPosition,
```
Comment (YuWernPong) [inline src/css.js]: cecar

File: src/css.js (hunk 2 above)
Comment (DedAnton) [inline src/css.js]: че за нах

**Pull request closed without merging**